### PR TITLE
suggest to separate kubernetes and slurm examples

### PR DESCRIPTION
**Pull request by KeitaW**
While reviewing PR #362, I noticed that combining different how-to guides for both schedulers (Slurm and Kubernetes) might be confusing for users. Typically, a user only needs guidance for one specific scheduler when following an example. This PR aims to discuss the possible direction of separating these guides into distinct subdirectories. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: 3.test_cases/16.pytorch-cpu-ddp/README.md [hunk 1]
```diff
@@ -6,242 +6,4 @@ Isolated environments are crucial for reproducible machine learning because they
...
-```
-
-
-Stop the training job:
-```bash
-kubectl delete -f ./fsdp.yaml
-```
-
-Note: Prior to running a new job, please stop any currently running or completed fsdp job.
+We will provide guide for both Slurm and Kubernetes but the conda example is only compatible with Slurm. Proceed to the subdirectory `slurm` or `kubernetes`.
```
Comment (KeitaW) [inline 3.test_cases/16.pytorch-cpu-ddp/README.md:9]:
```suggestion
 We will provide guides for both Slurm and Kubernetes. However, please note that the Conda example is only compatible with Slurm. For detailed instructions, proceed to the `slurm` or `kubernetes` subdirectory. ```

File: 3.test_cases/16.pytorch-cpu-ddp/kubernetes/README.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
+## 1. Preparation
+The guide assumes that you have the following:
+
+* An EKS cluster on AWS with x86-based CPU nodes, accessible via `kubectl`.
```
Comment (mhuguesaws) [inline 3.test_cases/16.pytorch-cpu-ddp/kubernetes/README.md:4]:
```suggestion
* An Amazon EKS cluster on AWS with x86-based CPU nodes, accessible via `kubectl`.
```

File: 3.test_cases/16.pytorch-cpu-ddp/kubernetes/README.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+* An FSx for Lustre persistent volume claim named `fsx-pv`, you can use an example from [here](https://github.com/aws-samples/aws-do-eks/tree/main/Container-Root/eks/deployment/csi/fsx), if you need to create one.
```
Comment (mhuguesaws) [inline 3.test_cases/16.pytorch-cpu-ddp/kubernetes/README.md:5]:
```suggestion
* An Amazon FSx for Lustre persistent volume claim named `fsx-pv`, you can use an example from [here](https://github.com/aws-samples/aws-do-eks/tree/main/Container-Root/eks/deployment/csi/fsx), if you need to create one.
```

File: 3.test_cases/16.pytorch-cpu-ddp/README.md [hunk 4]
```diff
@@ -6,242 +6,4 @@ Isolated environments are crucial for reproducible machine learning because they
...
-```
-
-
-Stop the training job:
-```bash
-kubectl delete -f ./fsdp.yaml
-```
-
-Note: Prior to running a new job, please stop any currently running or completed fsdp job.
+ We will provide guides for both Slurm and Kubernetes. However, please note that the Conda example is only compatible with Slurm. For detailed instructions, proceed to the `slurm` or `kubernetes` subdirectory. ```
```
Comment (mhuguesaws) [inline 3.test_cases/16.pytorch-cpu-ddp/README.md:9]:
```suggestion
 We provide guides for both Slurm and Kubernetes. However, please note that the Conda example is only compatible with Slurm. For detailed instructions, proceed to the `slurm` or `kubernetes` subdirectory. ```
```

File: 3.test_cases/16.pytorch-cpu-ddp/kubernetes/README.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+* Docker 
+
+We recommend that you setup a Kubernetes cluster using the templates in the architectures [directory](../../1.architectures). 
+
+
+## 3. Submit training job using Docker container
```
Comment (mhuguesaws) [inline 3.test_cases/16.pytorch-cpu-ddp/kubernetes/README.md:11]:
```suggestion
## 3. Submit training job using container
```

**Review by mhuguesaws — COMMENTED** [review]
Left comment. 
Overall good improvements on organization.

**Comment by iankouls-aws** [comment]
Organizing examples in orchestrator-specific subfolders is a good idea

**Pull request merged**